src/test/resources/features/textNegotiation.feature

```diff
@@ -5,6 +5,11 @@ Scenario: The text file exists
   Then the server should respond with status code 200 OK 
   And the server should respond with a message body of "This is a sample text file."
 
+Scenario: The text file exists and contains spaces in its name 
+  When a client makes a GET request to /file%20with%20space.txt
```

Comment:
Is this going to look for a file in my filesystem with that name? It doesn't appear to be included in this PR.

Reply:
It does look for a file with that name, and I accidentally forgot to include that file. I've added it now, and all tests pass.